JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/ViewModels/LoginViewModel.cs

```diff
@@ -10,13 +10,23 @@ namespace JhipsterXamarin.ViewModels
     public class LoginViewModel : MvxViewModel
     {
         private readonly IAuthenticationService _authenticationService;
+
+        public IAuthenticationService GetAuthenticationService()
```

Comment:
_authenticationService is also injected with mvvmcross you don't need to make it public nor create a getter in any situations

Reply:
He's not injected, look at the test. I need to do like this. I don't know of any other way.